share/picongpu/tests/CollisionsThermalisation/include/picongpu/param/grid.param

```diff
@@ -0,0 +1,106 @@
+/* Copyright 2013-2021 Axel Huebl, Rene Widera, Benjamin Worpitz, Pawel Ordyna
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Definition of cell sizes and time step. Our cells are defining a regular,
+ * cartesian grid. Our explicit FDTD field solvers define an upper bound for
+ * the time step value in relation to the cell size for convergence. Make
+ * sure to resolve important wavelengths of your simulation, e.g. shortest
+ * plasma wavelength and central laser wavelength both spatially and
+ * temporarily.
+ *
+ * **Units in reduced dimensions**
+ *
+ * In 2D3V simulations, the CELL_DEPTH_SI (Z) cell length
+ * is still used for normalization of densities, etc..
+ *
+ * A 2D3V simulation in a cartesian PIC simulation such as
+ * ours only changes the degrees of freedom in motion for
+ * (macro) particles and all (field) information in z
+ * travels instantaneous, making the 2D3V simulation
+ * behave like the interaction of infinite "wire particles"
+ * in fields with perfect symmetry in Z.
+ *
+ */
+
+#pragma once
+
+#include "picongpu/param/density.param"
+
+namespace picongpu
+{
+    namespace SI
+    {
+        /** Duration of one timestep
+         *
+         *  unit: seconds */
+        constexpr float_64 DELTA_T_SI = (1.0 / REFERENCE_OMEGA_SI) * 0.2 * 2.0 * PI;
```

Comment:
https://github.com/SmileiPIC/Smilei/blob/master/benchmarks/collisions/thermalisation_ei1.py
I thought that way it's easier to compare. `smilei` has a totally different approach to units so I guess it looks a bit weird in picongpu.

Reply:
Okay I get it. Could you perhaps make it:
```suggestion
        constexpr float_64 REFERENCE_TIME_SI = 2.0 * PI / REFERENCE_OMEGA_SI;
        constexpr float_64 DELTA_T_SI = 0.2 * REFERENCE_TIME_SI;
```
This would make clear how the numbers are related to each other without the need of extra comments.